libursa/src/ffi/cl/verifier.rs

```diff
@@ -45,6 +46,44 @@ pub extern "C" fn ursa_cl_verifier_new_proof_verifier(
     res
 }
 
+/// Add a common attribute to the proof verifier
+///
+/// # Arguments
+/// * `proof_builder` - Reference that contain proof verifier instance pointer.
+/// * `attribute_name` - Common attribute's name
+
+#[no_mangle]
+pub extern "C" fn ursa_cl_proof_verifier_add_common_attribute(
+    proof_verifier: *const c_void,
+    attribute_name: *const c_char,
```

Comment:
Again, we need to move away from this ASAP and replace it with some kind of serde marshalling so that the types flowing in/out of the Rust side are checked at compile and runtime.